docs/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 
 ## Unreleased
 ### Added
+* Added JsonFormat.
```

Comment:
Please link the PR in the change log entry.

Reply:
Is there a way to see the github issue number beforehand? Fixed it